dist2src/core.py

```diff
@@ -271,6 +273,21 @@ def dist_git_spec(self):
         )
         return self._dist_git_spec
 
+    @lru_cache()
```

Comment:
so that we would not load the file twice since it's not gonna change within one session

Reply:
Okay, I see now, this will cause this method to be run once during the lifetime of the `Dist2Src` object and return the cached return value for subsequent calls. Kinda not obvious, maybe it would worth mentioning in the doc-string.

Reply:
Since `@lru_cache` is a standard python decorator with terrible documentation, I'll add there a comment with what you just wrote since it's short and clear, thanks!